.github/CODEOWNERS

```diff
@@ -32,6 +32,9 @@ scripts/ @assignUser @majetideepak
 # Breeze 
 velox/experimental/breeze @dreveman
 
+# cuDF
+velox/experimental/cudf @bdice @karthikeyann @devavret
```

Comment:
fyi: in order to be eligible as codeowner you need write access to the repo

Reply:
Is that something we can request? @pedroerp might know more here.